frontend/src/components/reflection/subjectArea/subjectArea.test.tsx

```diff
@@ -0,0 +1,118 @@
+import { render, act, fireEvent } from '@testing-library/react'
+
+import { SubjectArea } from './subjectArea'
+
+const testTopics = [
+  {
+    id: 'element-1',
+    lable: '1 Fachinhalte',
+    describe:
+      'Die Lehrperson verfügt über fachwissenschaftliches und fachdidaktisches Wissen, versteht die Inhalte, Strukturen und zentralen Forschungsmethoden ihrer Fachbereiche und sie kann Lernsituationen schaffen, die die fachwissenschaftlichen und fachdidaktischen Aspekte für die Lernenden bedeutsam machen.',
+    subListItems: [
+      {
+        id: 'element-1.1',
+        lable: '1.1 Fachwissen und Didaktik',
+        done: false,
+      },
+      {
+        id: 'element-1.2',
+        lable: '1.2 Fachwissen und Didaktik',
+        done: true,
+      },
+    ],
+  },
+  {
+    id: 'element-2',
+    lable: '2 Entwicklungsprozesse',
+    describe:
+      'Die Lehrperson versteht, wie Kinder und Erwachsene lernen und sich entwickeln, und sie kann Lerngelegenheiten und Lernwege anbieten, welche die kognitive, soziale und persönliche Entwicklung unterstützen.',
+    subListItems: [
+      {
+        id: 'element-2.1',
+        lable: '2.1 Entwicklung der Lernenden',
+        done: true,
+      },
+      {
+        id: 'element-2.2',
+        lable: '2.2 Erfahrungen und Vorwissen',
+        done: true,
+      },
+    ],
+  },
+]
```

Comment:
Can't you load the real data? and select 2 elements?

Reply:
This are real-Data but only a subset and not the complete one. If we have the full data available we can replace it,